photonlib-java-examples/src/dev/native/cpp/main.cpp

```diff
@@ -0,0 +1,8 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2020 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+int main() {}

```

Comment:
I'd blindly cloned this from the wpi example, but it wasn't clear to me if it was actually necessary?

Reply:
It is. If there is no cpp file, the C++ tasks are skipped and the dylibs are not copied to the correct dir